src/neo/UIntBase.cs

```diff
@@ -1,13 +1,15 @@
 using Neo.IO;
 using System;
 using System.IO;
+using System.Runtime.InteropServices;
 
 namespace Neo
 {
     /// <summary>
     /// Base class for little-endian unsigned integers. Two classes inherit from this: UInt160 and UInt256.
     /// Only basic comparison/serialization are proposed for these classes. For arithmetic purposes, use BigInteger class.
     /// </summary>
+    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Ansi, Pack = 1)]
```

Comment:
This is not needed and should be removed.

Reply:
We need this line, or replace by `[StructLayout(LayoutKind.Explicit, Size = 0)]`,  as `UIntBase` is the base class which also needs this, otherwise the ut can't pass.

Reply:
I removed `UIntBase`.

Reply:
I removed `UIntBase`.

Reply:
We can remove it if you want, it's not used

Reply:
I am not sure if we need it, maybe the compiler optimizations will need it